applications/dashboard/models/class.usermodel.php

```diff
@@ -2932,18 +2932,24 @@ public function saveIP($userID, $IP, $dateUpdated = false) {
 
     /**
      * Updates visit level information such as date last active and the user's ip address.
-     *
-     * @param int $UserID
-     * @param string|int|float $ClientHour
+     * @param $UserID
+     * @param int|float $ClientHour
```

Comment:
$ClientHour can be a float.
https://en.wikipedia.org/wiki/List_of_UTC_time_offsets#UTC.2B13:45 for example.